Gemfile

```diff
@@ -10,12 +10,13 @@ gem 'dotenv-rails'
 gem 'enum_help'
 gem 'figaro'
 gem 'foreman', require: false
-gem 'hashie'
+gem 'hashie', '~> 3.5', '>= 3.5.5'
```

Comment:
So it looks like this means, 
highest version of 3.5 and higher than or equal to 3.5.5

I think its the same as `~> 3.5.5`